tests/integration/references/belongs-to-test.js

```diff
@@ -230,7 +230,7 @@ test("push(promise)", function(assert) {
     push = familyReference.push(deferred.promise);
   });
 
-  assert.ok(push.then, 'BelongsToReference.push returns a promise');
+  assert.ok(push instanceof Ember.RSVP.Promise, 'BelongsToReference.push() returns a promise');
```

Comment:
yes but what I am trying to test is that we are not returning `PromiseProxyObject`. If you have a better idea on how to test this...